src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -144,6 +144,9 @@ bool FrontEnd::supported_impl(const std::vector<ov::Any>& variants) const {
     else if (variants[0].is<GraphIterator::Ptr>()) {
         // this is used for OpenVINO with TensorFlow Integration
         return true;
+    } else if (variants[0].is<std::shared_ptr<IGraphIterator>>()) {
+        // this is used for Python implemetation of GraphIterator
+        return true;
```

Comment:
Looks strange that we unwrap, in principle, the same `GraphIterator` objects in different lines.

Reply:
I moved GraphIterator to common FE, so IGraphIterator is no longer needed. Removed this code.